### DISCUSSION TITLE
v2.0.12 API docs

### DISCUSSION THREAD

**Discussion by dreautall** [firefly-iii/firefly-iii#8270] [General]
Hi there! Any ETA on the v2.0.12 API doc swagger update?

Thank you for all the quick updates and even integrating some of my wishes! 😄

**Comment by JC5** [comment]
I'm terribly behind with those, no ETA yet, sorry!

**Reply by dreautall** [reply]
Hi, I see that it has now been updated. However, unfortunately I can't find the new feature from https://github.com/firefly-iii/firefly-iii/issues/8022 in there - did you maybe miss that? Thank you!
